packages/blocks/src/__internal__/rich-text/rich-text-operations.ts

```diff
@@ -276,8 +286,15 @@ export function handleLineStartBackspace(page: Page, model: ExtendedModel) {
             page.deleteBlock(model);
           }
         });
+      } else {
+        const richText = getRichTextByModel(model);
+        if (richText) {
+          page.captureSync();
+          page.deleteBlock(model);
```

Comment:
Deleting this starting block may bring a cursor blind zone. Instead, `model.text.delete(0, model.text.length)` can avoid this.

Also, IMO we need a `matchFlavours(model, ['affine:list', 'affine:paragraph'])` guard _here,_ so that other block flavours (types) won't be affected.